host/workflowidratelimit_test.go

```diff
@@ -23,14 +23,86 @@
 package host
 
 import (
+	"flag"
+	"testing"
+
 	"github.com/pborman/uuid"
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+	"github.com/uber/cadence/common/clock"
+	"github.com/uber/cadence/common/dynamicconfig"
+	"github.com/uber/cadence/common/persistence"
+	pt "github.com/uber/cadence/common/persistence/persistence-tests"
 
 	"github.com/uber/cadence/common"
 	"github.com/uber/cadence/common/types"
 )
 
-func (s *IntegrationSuite) TestWorkflowIDSpecificRateLimits() {
+func TestWorkflowIDRateLimitIntegrationSuite(t *testing.T) {
+	flag.Parse()
```

Comment:
why do you need to parse command-line?

Reply:
If it is required, it deserves a comment since it is very unexpected for test to parse cmd-line.

Reply:
Added a comment